### PR TITLE
Add clear browsing data preference

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch means adding clear browsing data preference. It's on the
banana extension setting and will open clear browsing data fragment when
a user clicks this one. Also, this preference is available on the toolbar.

Refs: #768

### CODE REVIEW

File: res/values-ko/strings.xml [hunk 1]
```diff
@@ -49,5 +49,6 @@
     <string name="secure_login_summary">웹 페이지에 로그인을 할 때, 저장된 사용자 비밀번호를 사용하는 경우 추가 인증을 사용합니다 (예: 생체 인증, 패턴, PIN 등)</string>
     <string name="toolbar_editor">도구모음 편집하기</string>
     <string name="user_interface">사용자 인터페이스</string>
+    <string name="user_privacy">사용자 개인정보 보호</string>
```
Comment (romandev) [inline res/values-ko/strings.xml:52]: 사용자 기록

File: res/values/strings.xml [hunk 2]
```diff
@@ -49,5 +49,6 @@
     <string name="secure_login_summary">Use additional authentications to autofill stored credentials data securely when you try to sign-in on web sites (e.g. Biometrics, Pattern, PIN and so on)</string>
     <string name="toolbar_editor">Toolbar Editor</string>
     <string name="user_interface">User Interface</string>
+    <string name="user_privacy">User Privacy</string>
```
Comment (romandev) [inline res/values/strings.xml:52]: "user_data"

File: res/values-ko/strings.xml (hunk 1 above)
Comment (jdragonbae) [reply res/values-ko/strings.xml]: Thanks. Done.

File: res/values/strings.xml (hunk 2 above)
Comment (jdragonbae) [reply res/values/strings.xml]: Done.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**